src/components/product/PriceOfferModal/index.tsx

```diff
@@ -0,0 +1,154 @@
+import { Input, Divider, Radio, IconButton } from '@offer-ui/react'
+import type { ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type { OfferForm, PriceOfferModalProps } from './types'
+import { TRADE_METHOD } from '@constants'
+
+const MOCK_CITY = [
+  { code: 1, name: 'Seoul' },
+  {
+    code: 2,
+    name: 'Deajeon'
+  },
+  {
+    code: 3,
+    name: 'Incheon'
+  }
+]
+
+const initialOfferForm = {
+  tradeMethod: 0,
+  tradeLocation: {
+    city: 0,
+    country: 0,
+    town: 0
+  }
+}
+
+export const PriceOfferModal = ({
+  handleClickOffer,
+  ...props
+}: PriceOfferModalProps): ReactElement => {
+  const [offerForm, setOfferForm] = useState<OfferForm>(initialOfferForm)
+  const isTradeDirect = offerForm.tradeMethod !== 8
```

Comment:
```suggestion
  const isTradeDirect = offerForm.tradeMethod !== 8
```

@price-offer/offer-fe  8, 4와 같은 매직 리터럴도 논의 후에 수정되면 좋을 것 같습니다